mapeditor/mainwindow.cpp

```diff
@@ -38,9 +38,12 @@
 #include "mapsettings.h"
 #include "playersettings.h"
 #include "validator.h"
+#include "resourceExtractor/ResourceConverter.h"
 
 static CBasicLogConfigurator * logConfig;
 
+ExtractionOptions extractionOptions;
```

Comment:
please consider alternatives to having global variable, maybe something like that
```
void MainWindow::parseCommandLine(ExtractionOptions & ops)
{
...
}

MainWindow::MainWindow(QWidget * parent): ...
{
    ...
    ExtractionOptions extractionOptions;
    parseCommandLine(extractionOptions);
...
    preInitDLL(::console, false, extractionOptions.extractArchives);
}
```

Reply:
Done like suggested.